Cython/Compiler/Scanning.py

```diff
@@ -274,6 +284,9 @@ def get_lines(self, encoding=None, error_handling=None):
     def get_description(self):
         return self.name
 
+    def get_relative_path(self):
+        return Path(self.get_description())
+
     get_error_description = get_description
 
     def get_filenametable_entry(self):
```

Comment:
We previously used `"stringsource>"` here and you changed it to `self.name`. Why?

Reply:
I'm not sure how the previous code worked in this regard. If you would use `stringsource>` here, then above in the `Compilor/<x>Nodes` you would call `Path("stringsource>")` which doesn't look right. I suspect that it got filtered out by the `isabs` check, which then falls back to the description > name.